JeffShepherd/Rancid-Tomatillos
# Feature/style app

## What does this PR do?

- Add css to style and position elements in all components.
- Add normalize css file to reset default css values.


## How should it be tested in the UI?

- cd into Rancid-Tomatillos repo, run npm start
- On page load, all elements should appear evenly spaced
- Home button should have a white outline hover effect
- On the details page, the background should be the movie backdrop image, and the relevant details should be evenly spaced in the bottom-left corner
